Tests/web3swiftTests/localTests/ABIDecoderSliceTests.swift

```diff
@@ -74,17 +72,15 @@ final class ABIDecoderSliceTests: XCTestCase {
         let erc20_balanceof = try EthereumContract(Web3.Utils.erc20ABI).methods["balanceOf"]!.first!
 
         /// Act
-        guard let decodedData = contract.decodeReturnData("tryAggregate", data: data) else {
-            throw Web3Error.processingError(desc: "Can not decode returned parameters")
-        }
+        let decodedData = try contract.decodeReturnData("tryAggregate", data: data)
 
         guard let returnData = decodedData["returnData"] as? [[Any]] else {
             throw Web3Error.dataError
         }
         var resultArray = [BigUInt]()
         for i in 0..<2 {
             guard let data = returnData[i][1] as? Data,
-                  let balance = erc20_balanceof.decodeReturnData(data)["0"] as? BigUInt else {
+                  let balance = try? erc20_balanceof.decodeReturnData(data)["0"] as? BigUInt else {
```

Comment:
```suggestion
                  let balance = try erc20_balanceof.decodeReturnData(data)["0"] as? BigUInt else {
```

Reply:
When using the `tryAggregate` method of the [multicall2](https://etherscan.io/address/0x5ba1e12693dc8f9c48aad8770482f4739beed696#code
) contract to query in batches, if the `requireSuccess` parameter is false, the contract will ignore errors in the sub-items and fill zero bytes in the `returnData`. In this case, we should use `try?` to ignore the sub-items decoding error